.github/workflows/test.yml

```diff
@@ -80,7 +80,7 @@ jobs:
       # Configure test runner
       - uses: game-ci/unity-test-runner@v4.3.1
         id: testRunner
-        timeout-minutes: 60
+        timeout-minutes: 120
```

Comment:
Could we go with 80, to just fail quicker, should be still enough for first run. What do you think?